README.md

```diff
@@ -68,41 +68,50 @@ actualEffects
 
 ```js
 // favSaga.js
-function* favSagaWorker(action) {
+function* retryFavSagaWorker(action) {
   const { itemId } = action.payload
   const { token, user } = yield select(getGlobalState)
 
-  try {
-    const response = yield call(favItem, itemId, token)
-    const json = yield response.json()
-    yield put(sucessfulFavItemAction(json, itemId, user))
-  } catch (e) {
-    yield put(receivedFavItemErrorAction(e, itemId))
+  let attempt = 0
+  while (attempt++ < 5) {
+    try {
+      const response = yield call(favItem, itemId, token)
+      const json = yield response.json()
+      yield put(sucessfulFavItemAction(json, itemId, user))
+      break
+    } catch (e) {
+      yield put(receivedFavItemErrorAction(e, itemId))
+      yield call(delay, 2000)
+    }
```

Comment:
I don't think I want this to be the main example on the `README.md`. I'd like to stick with the simplest but reasonable example of a saga for testing, so the purpose of the library can be . I don't think the `retry` pattern is very commonly-used, after all.

But for sake of convenience, I think I'll accept this PR as is because we want everything else it contains in. I'll just change the `README` back later, if there are no objections :-)